x-pack/plugins/fleet/common/openapi/components/schemas/fleet_settings_enrollment_response.yaml

```diff
@@ -23,6 +23,12 @@ properties:
               type: string
             download_source_id:
               type: string
+            space_ids:
```

Comment:
thanks for adding this 👍

Reply:
👍 the spec for this API isn't actually compiled due to it being an internal route, but figured I'd keep it up-to-date anyway